app/src/main/java/live/teekamsuthar/mutify/MainActivity.java

```diff
@@ -781,6 +781,9 @@ private void isChineseBrand(){
                 intent = new Intent().setComponent(new ComponentName("com.asus.mobilemanager", "com.asus.mobilemanager.entry.FunctionActivity")).setData(
                         Uri.parse("mobilemanager://function/entry/AutoStart"));
                 break;
+            case "samsung":
+                intent = new Intent().setComponent(new ComponentName("com.samsung.android.lool", "com.samsung.android.sm.ui.battery.BatteryActivity"));
+                break;
             default:
```

Comment:
Thanks for the inclusion! Do you have any of these devices that you can test on? I only have Motorola, and I want to make sure everything's working as expected before pushing this as an update.

Reply:
I have tested it on Xiaomi  and Samsung and it's working fine

Reply:
Is something wrong with my Pull Request?

Reply:
Thanks for the update! I'm working on a new, more aggressive pop-up for "Device Broadcast Status" and once I finish this, I'll try to integrate your changes to that same pop up. So, the pull request will be merged soon, Thank you so much for your contribution. :)

Reply:
Thank You